### PR TITLE
[19.01] Release notes for 19.01.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: doc/source/releases/19.01_announce.rst [hunk 1]
```diff
@@ -3,8 +3,71 @@
...
+  to serve as a file editor for such workflows (`Pull Request 7019`_). The workflow editor features
+  more helpful labels on inputs and outputs (`Pull Request 6770`_) and important fixes including
+  logic for connecting collections (`Pull Request 6732`_) and dealing with missing tools
+  (`Pull Request 7227`_, `Pull Request 7258`_). The workflow run form now deals with default
+  values more correctly (`Pull Request 7157`_).
+
+**Support for Singularity Biocontainers**
+  Galaxy can now be configured to fetch `Biocontainers <https://biocontainers.pro/>`__ automatically
+  for `Singularity <https://singularity.lbl.gov/>`__ enabled job destinations. Previously this
+  would happen automatically for Docker containers but not Singularity containers. Thanks to
```
Comment (mvdbeek) [inline doc/source/releases/19.01_announce.rst]: There's also support for explicit docker container annotation. There should be feature parity between docker and singularity now.

File: doc/source/releases/19.01.rst [hunk 2]
```diff
@@ -0,0 +1,810 @@
...
+* Add datatype sniff button to edit dataset attributes page
+  (thanks to `@qiagu <https://github.com/qiagu>`__).
+  `Pull Request 7018`_
+* Overhaul handler assignment methods.
+  `Pull Request 7129`_, `Pull Request 7091`_
+* Add tool for sending dataset to the cloud.
+  `Pull Request 6583`_
+* Infrastructure/API Support for composite uploads/downloads.
+  `Pull Request 6844`_
+* Improvements to tool testing including aloowing tests without admin API keys.
```
Comment (mvdbeek) [inline doc/source/releases/19.01.rst]:
```suggestion
* Improvements to tool testing including allowing remote tests and tests without admin API keys.
```

File: doc/source/releases/19.01.rst [hunk 3]
```diff
@@ -0,0 +1,810 @@
...
+* Replace display applications mako with vue component
+  `Pull Request 7097`_
+* Run singularity in silent mode
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 7104`_
+* Eliminate unused code block in `output_collect`.
+  `Pull Request 7109`_
+* Add `to_dict()` to ObjectStores.
+  `Pull Request 7110`_
+* Enable tagginga and annotation of jobs regardless of state.
```
Comment (mvdbeek) [inline doc/source/releases/19.01.rst]:
```suggestion
* Enable tagging and annotation of jobs regardless of state.
```

**Comment by nsoranzo** [comment]
Any thoughts on including https://github.com/galaxyproject/galaxy/issues/7260 ?

**Comment by jmchilton** [comment]
@nsoranzo I'm -.9 on that, see comments on that issue.

**Comment by martenson** [comment]
@nsoranzo I think this is too late to include such a commitment in our release notes without a broad consensus amongst the committers.

**Pull request merged**